spec/support/corpus.rb

```diff
@@ -124,6 +116,42 @@ def checkout
 
     private
 
+      MissingExpectedException = Class.new(StandardError)
```

Comment:
Does defining this here actually make it private? I don't think it has any effect. I would tend to define constants near the top of the namespace, just to give a consistent layout, and to not provide a false signal that it is private.

Also I did a quick search and I don't see this constant used anywhere in this diff. Is it used elsewhere?